cdk/domino_cdk/config_template.yaml

```diff
@@ -24,31 +40,36 @@ route53:
 eks:
   version: 1.19
   private_api: true
+  max_nodegroup_azs: 1
   global_node_labels:
     dominodatalab.com/domino-node: "true"
-  managed_nodegroups:
+  managed_nodegroups: {}
+  nodegroups:
     platform:
-      spot: false
-      disk_size: 20
+      gpu: false
+      ssm_agent: true
+      disk_size: 100
+      # key_name: some-aws-keypair
       min_size: 1
-      max_size: 1
-      desired_size: 1
+      max_size: 10
       # machine_image:
       #   ami_id: ami-xxxxx
       #   user_data: |
-      #     data goes here
+      #     #!/bin/bash
+      #     echo ${ClusterName}
       instance_types:
         - m5.2xlarge
       labels:
         dominodatalab.com/node-pool: platform
       tags:
         dominodatalab.com/node-pool: platform
     compute:
-      spot: false
+      gpu: false
+      ssm_agent: true
       disk_size: 100
-      min_size: 1 # min 0 not supported w/managed node groups
+      # key_name: some-aws-keypair
+      min_size: 0
```

Comment:
12:13:38 AM | CREATE_FAILED        | AWS::EKS::Nodegroup                   | eks/Nodegroupcompute-0
minSize can't be zero or negative

Reply:
I don't think your config is actually based on this template. You'll note I moved all the default nodegroups under the unmanaged nodegroups, but your error suggests they're in the managed ones.

Reply:
Correct